components/markets/SellFullSetForm.tsx

```diff
@@ -52,10 +52,10 @@ const SellFullSetForm = ({
     fee,
   } = useExtrinsic(
     () => {
-      if (isRpcSdk(sdk)) {
+      if (isRpcSdk(sdk) && amount && amount !== "") {
         return sdk.api.tx.predictionMarkets.sellCompleteSet(
```

Comment:
getting the same error here as in the buy full set. 

Also, not sure if we want this behaviour but I'm able to type `e` within the input boxes. Assuming that's math related.